sdk/tables/azure-data-tables/src/test/java/com/azure/data/tables/TableAsyncClientTest.java

```diff
@@ -927,6 +953,8 @@ public void generateSasTokenWithAllParameters() {
 
     @Test
     public void canUseSasTokenToCreateValidTableClient() {
+        Assumptions.assumeFalse(IS_COSMOS_TEST, "SAS Tokens are not supported for Cosmos endpoints.");
```

Comment:
A Table-level SAS is supported by Cosmos - or are those separate tests?

Reply:
You are right, this is supported on the Table level by Cosmos. I will remove the assumption.

Reply:
Though I can't help but wonder why these tests fail for Cosmos and not for Storage, since we use the same logic to generate both SAS tokens and we don't really need to do anything special for Cosmos.